.gitignore

```diff
@@ -1,6 +1,11 @@
 # Build Directory
 DerivedData
+
 # User Data
 *.pbxuser
 xcuserdata
 
+# Docs
```

Comment:
Hmmm. I rebased against master, so this got picked up. No worries. When we rebase back, this change will disappear.